diff -ruN usr/lib/ruby/gems/1.8/gems/mongrel-1.0.1/bin/mongrel_rails usr-new/lib/ruby/gems/1.8/gems/mongrel-1.0.1/bin/mongrel_rails
--- usr/lib/ruby/gems/1.8/gems/mongrel-1.0.1/bin/mongrel_rails	2007-08-23 16:39:37.000000000 -0400
+++ usr-new/lib/ruby/gems/1.8/gems/mongrel-1.0.1/bin/mongrel_rails	2007-08-23 16:42:15.000000000 -0400
@@ -9,7 +9,10 @@
 require 'mongrel'
 require 'mongrel/rails'
 require 'etc'
-require 'cgi_multipart_eof_fix' rescue nil
+begin
+  require 'cgi_multipart_eof_fix'
+rescue LoadError
+end
 
 module Mongrel
   class Start < GemPlugin::Plugin "/commands"
diff -ruN usr/lib/ruby/gems/1.8/specifications/mongrel-1.0.1.gemspec usr-new/lib/ruby/gems/1.8/specifications/mongrel-1.0.1.gemspec
--- usr/lib/ruby/gems/1.8/specifications/mongrel-1.0.1.gemspec	2007-08-23 16:39:38.000000000 -0400
+++ usr-new/lib/ruby/gems/1.8/specifications/mongrel-1.0.1.gemspec	2007-08-23 16:41:41.000000000 -0400
@@ -16,5 +16,4 @@
   s.add_dependency(%q<daemons>, [">= 1.0.3"])
   s.add_dependency(%q<fastthread>, [">= 0.6.2"])
   s.add_dependency(%q<gem_plugin>, [">= 0.2.2"])
-  s.add_dependency(%q<cgi_multipart_eof_fix>, [">= 1.0.0"])
 end
